src/mumble/AudioInput.cpp

```diff
@@ -943,6 +944,8 @@ int AudioInput::encodeCELTFrame(short *psSource, EncodingOutputBuffer &buffer) {
 	return len;
 }
 
+static const int MUTE_CUE_DELAY = 5000;  // in ms
```

Comment:
I think I would turn this into a static member variable in the `AudioOutput` class. Also I would suggest to use `constexpr` instead of `const` for this kind of stuff.

Reply:
Yeah sorry, I meant `AudioInput`